include/pybind11/detail/common.h

```diff
@@ -479,6 +479,8 @@ PYBIND11_WARNING_POP
         }
 
 \endrst */
+PYBIND11_WARNING_PUSH
+PYBIND11_WARNING_DISABLE_CLANG("-Wgnu-zero-variadic-macro-arguments")
```

Comment:
Just double-checking: is this needed, even with `-std=c++20`?

Reply:
Yeah, for some reason Clang produces a warning here. GCC is fine, but Clang complains. It produces a warnings saying this is a C++20 feature on newer compilers if set less than C++20, so no idea why the old warning triggers in C++20 mode. I think it's a bug.